app/uk/gov/hmrc/excisemovementcontrolsystemapi/services/NrsServiceNew.scala

```diff
@@ -0,0 +1,129 @@
+/*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.excisemovementcontrolsystemapi.services
+
+import org.apache.pekko.Done
+import play.api.Logging
+import uk.gov.hmrc.auth.core._
+import uk.gov.hmrc.auth.core.retrieve._
+import uk.gov.hmrc.auth.core.retrieve.v2.Retrievals
+import uk.gov.hmrc.excisemovementcontrolsystemapi.connectors.NrsConnectorNew
+import uk.gov.hmrc.excisemovementcontrolsystemapi.models.auth.ParsedXmlRequest
+import uk.gov.hmrc.excisemovementcontrolsystemapi.models.nrs._
+import uk.gov.hmrc.excisemovementcontrolsystemapi.repository.NRSWorkItemRepository
+import uk.gov.hmrc.excisemovementcontrolsystemapi.repository.model.NrsSubmissionWorkItem
+import uk.gov.hmrc.excisemovementcontrolsystemapi.services.NrsService.nonRepudiationIdentityRetrievals
+import uk.gov.hmrc.excisemovementcontrolsystemapi.utils.{DateTimeService, EmcsUtils, NrsEventIdMapper}
+import uk.gov.hmrc.http.{Authorization, HeaderCarrier, InternalServerException}
+import uk.gov.hmrc.mongo.workitem.{ProcessingStatus, WorkItem}
+
+import javax.inject.{Inject, Singleton}
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class NrsServiceNew @Inject() (
+  override val authConnector: AuthConnector,
+  nrsConnectorNew: NrsConnectorNew,
+  nrsWorkItemRepository: NRSWorkItemRepository,
+  dateTimeService: DateTimeService,
+  emcsUtils: EmcsUtils,
+  nrsEventIdMapper: NrsEventIdMapper,
+  correlationIdService: CorrelationIdService
+)(implicit ec: ExecutionContext)
+    extends AuthorisedFunctions
+    with Logging {
+
+  def makeNrsWorkItemAndAddToRepository(
```

Comment:
I don't love this name, if only because it's wordy and implementationy

Maybe something simpler - scheduleNrsSubmission?

Reply:
Or even just scheduleNrs